admin-dev/themes/default/scss/controllers/_themes.scss

```diff
@@ -58,6 +58,21 @@
   padding-top: 0;
 }
 
+#psthemecusto {
```

Comment:
@tblivet I'll approve but this should be moved to the module's code some other day